### DISCUSSION TITLE
Link the skin compartment to the Venous blood compartment in whole body system

### DISCUSSION THREAD

**Discussion by jonathanlsn** [Open-Systems-Pharmacology/Forum#909] [Questions & Problems]
Hello @abdullahhamadeh,

I am working on the permeation of compounds through the skin to venous blood plasma so I used the skin_permeation model for my simulations. I followed the instructions to connect DERMAL_APPLICATION_AREA|PKSim_link to VenousBlood|Plasma as it's said in the user guide and then I merged the whole_body_simulation.pkml with the skin_simulation.pkml. Then, when I run the simulation and seeing the kinetic of permeant amount in VenousBlood Plasma I obtain exactly the same results as the whole_body_simulation I made in PK-Sim. I can see the permeant kinetic through skin layers, but I have the feeling there is no contribution from the DERMAL_APPLICATION_AREA compartment into the calculation of permeant amount in the plasma.

Could you please explain why the merging step between the DERMAL_APPLICATION_AREA and the whole body doesn't seem to affect the kinetic results into the plasma ?

Thank you in advance

**Comment by abdullahhamadeh** [comment]
Hi @jonathanlsn 
Did you set in_vivo = 0?  It needs to be in_vivo = 1

**Reply by jonathanlsn** [reply]
The in_vivo value in skin compartment is set to 1.
